tests/playwright/src/runner/runner-options.ts

```diff
@@ -0,0 +1,58 @@
+/**********************************************************************
+ * Copyright (C) 2023-2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+export class RunnerOptions {
+  public readonly _profile: string;
+  public readonly _customFolder: string;
+  public readonly _openDevTools: string;
+  public readonly _autoUpdate: boolean;
+  public readonly _autoCheckUpdates: boolean;
+  public readonly _extesionsDisabled: string[];
+
+  constructor({
+    profile = '',
+    customFolder = 'podman-desktop',
+    openDevTools = 'none',
+    autoUpdate = true,
+    autoCheckUpdates = true,
+    extesionsDisabled = [],
+  }: {
+    profile?: string;
+    customFolder?: string;
+    openDevTools?: string;
+    autoUpdate?: boolean;
+    autoCheckUpdates?: boolean;
+    extesionsDisabled?: string[];
+  } = {}) {
+    this._profile = profile;
+    this._customFolder = customFolder;
+    this._openDevTools = openDevTools;
+    this._autoUpdate = autoUpdate;
+    this._autoCheckUpdates = autoCheckUpdates;
+    this._extesionsDisabled = extesionsDisabled;
+  }
+
+  public createSettingsJson(): string {
+    return JSON.stringify({
+      'preferences.OpenDevTools': this._openDevTools,
+      'extensions.autoUpdate': this._autoUpdate,
+      'extensions.autoCheckUpdates': this._autoCheckUpdates,
+      'extensions.disabled': this._extesionsDisabled,
+    });
```

Comment:
I also need `podman.binary.path` as a parameterould you please add it?

Reply:
added